### PR TITLE
New now mobile

### PR DESCRIPTION
**Pull request by Vadim-Kostetskyi**

### CODE REVIEW

File: src/components/NewNowMobile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            />
+          </SwiperSlide>
+        ))}
+        <div className={styles.wrapperArrows}>
+          <button
+            ref={node => setPrevEl(node)}
+            className={activeIndex ? styles.itemArrow : styles.invisible}
+          >
+            <ArrowSwiperCard
+              className={`${styles.arrow} ${styles.arrowPrev}`}
```
Comment (denred) [inline src/components/NewNowMobile/index.tsx:47]: consider using a classnames or clsx utility library

File: src/components/NewNowMobile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              sizes={sizes}
+              isMobile={true}
+              image={images[0]}
+            />
+          </SwiperSlide>
+        ))}
+        <div className={styles.wrapperArrows}>
+          <button
+            ref={node => setPrevEl(node)}
+            className={activeIndex ? styles.itemArrow : styles.invisible}
```
Comment (denred) [inline src/components/NewNowMobile/index.tsx:44]: you have similar logic repeated, consider using the function

File: src/components/ProductCard/index.tsx [hunk 3]
```diff
@@ -8,17 +8,25 @@ export interface ProductCardProps {
...
   productName,
   sizes,
   images,
+  isMobile,
+  image,
 }) => (
   <div className={styles.productCard}>
-    <ProductImageSwiper images={images} productName={productName} />
+    {isMobile ? (
+      <img src={image} alt="productName" className={styles.image} />
```
Comment (denred) [inline src/components/ProductCard/index.tsx:26]: {productName}

File: src/components/NewNowMobile/index.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  const combinedClassName = clsx(styles.arrow, styles.arrowPrev);
+
+  const setSwiperIndex = useCallback(
+    ({ activeIndex }: { activeIndex: number }) => {
+      setActiveIndex(activeIndex);
+    },
+    [setActiveIndex],
+  );
+
+  const arrowClassName = (arrowPrev?: boolean) => {
```
Comment (sashaK3) [inline src/components/NewNowMobile/index.tsx:28]: wrap into `useMemo`

File: src/components/ProductImageSwiper/index.tsx [hunk 5]
```diff
@@ -6,12 +6,14 @@ import styles from './index.module.scss';
 
 export interface ProductImageSwiperProps {
   productName: string;
-  images: string[];
+  images?: string[];
+  // wrapperClassName: string;
```
Comment (sashaK3) [inline src/components/ProductImageSwiper/index.tsx:10]: delete

**Review by sashaK3 — APPROVED** [review]

**Pull request merged**